crankshaft-docker/src/service.rs

```diff
@@ -180,9 +193,6 @@ impl Service {
                     break (container_id, exit_code.unwrap());
                 }
                 Some(TaskState::COMPLETE) => {
-                    // Notify that the task has started (and has already completed)
-                    started();
```

Comment:
I think this also will need to handle jobs that go immediately to the `COMPLETE` state, right? What if we skip the other states?

Reply:
Oh, nevermind, I see it below.